samples/msal-browser-samples/HybridSample/views/client-redirect.hbs

```diff
@@ -6,44 +6,62 @@
     const msalInstance = new msal.PublicClientApplication({
         auth: {
             clientId: "{{clientId}}",
-            redirectUri: "http://localhost:3000/auth/client-redirect",
+            redirectUri: "https://localhost:3000/auth/client-redirect",
             authority: "{{authority}}"
+        },
+        system: {
+            loggerOptions: {
+                loggerCallback: (loglevel, message, containsPii) => {
+                    console.log(message);
+                },
+                piiLoggingEnabled: false,
+                logLevel: msal.LogLevel.Verbose,
+            },
+            allowNativeBroker: true
         }
-    })
+    });
+    
+    // Initialize native broker if available
+    msalInstance.initialize().then(() => {
 
-    // Only handle hash when in top frame
-    if (window === window.top) {
-        msalInstance.handleRedirectPromise()
-            .then(result => {
-                if (result) {
-                    console.log('MSAL: Returning from login');
-                    return result;
-                }
-                
-                const sid = "{{sid}}";
-                const code = "{{code}}";
-                const preferredUsername = "{{preferredUsername}}";
-                const loginHint = "{{{loginHint}}}";
+        // Only handle hash when in top frame
+        if (window === window.top) {
+            msalInstance.handleRedirectPromise()
+                .then(result => {
+                    if (result) {
+                        console.log('MSAL: Returning from login');
+                        console.log(result);
+                        return result;
+                    }
+                    
+                    const sid = "{{sid}}";
+                    const code = "{{code}}";
+                    const preferredUsername = "{{preferredUsername}}";
+                    const loginHint = "{{{loginHint}}}";
+                    
+                    const fromNativeBroker = "{{fromNativeBroker}}";
+                    const account = {{{account}}};
 
-                const scopes = [ "user.read" ];
-                const tokenQueryParameters = {
-                    dc: "ESTS-PUB-WUS2-AZ1-FD000-TEST1",
-                    hybridspa: "true"
-                }
+                    const scopes = [ "user.read" ];
+                    const tokenQueryParameters = {
+                        dc: "ESTS-PUB-WUS2-AZ1-FD000-TEST1",
```

Comment:
Test slice still needed?

Reply:
I think this is still needed to point to the hybrid test slice in ESTS. Should be removed in the future.